recovery/elliptics_recovery/dc_recovery.py

```diff
@@ -412,7 +420,9 @@ def recover(ctx):
                                  net_thread_num=4,
                                  io_thread_num=24,
                                  remotes=ctx.remotes)
+    ServerSendRecovery(ctx, node).recover()
```

Comment:
Why doesn't result of `ServerSendRecovery().recover()` affect to final result of whole recovery? What if it is failed and nothing can be recovered by `WindowDC()`.